pom.xml

```diff
@@ -19,25 +19,18 @@
 
     <logback-version>1.1.2</logback-version>
     <scala-maven-plugin-version>3.2.1</scala-maven-plugin-version>
-    <sbv>2.11</sbv>
-    <scala-version>${sbv}.7</scala-version>
-    <jodatime-version>2.3</jodatime-version>
-    <specs2-version>2.3.12</specs2-version>
-    <commons-codec-version>1.9</commons-codec-version>
-    <commons-lang-version>2.6</commons-lang-version>
-    <slf4j-version>1.7.7</slf4j-version>
-    <commons-io-version>2.4</commons-io-version>
-    <cglib-version>3.1</cglib-version>
-    <rundeck-version>3.3.18-20220118</rundeck-version>
+    <sbv>3.7</sbv>
+    <scala-version>${sbv}.1</scala-version>
+    <rundeck-version>4.0.0-20220322</rundeck-version>
   </properties>
   <build>
     <plugins>
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
I'm not really sure this plugin is actuelly required

Reply:
we don't build java here, the scala plugins seems enough (https://www.baeldung.com/scala/maven for example)